README.md

```diff
@@ -63,20 +67,16 @@ poetry add <package-name>
 
 ### Loading the Data
 
-The dataset is stored in a private repository on HuggingFace. It can easily be loaded as
-
-Begin by installing the library, if not already installed:
-
-```bash
-pip install huggingface_hub
-```
+The dataset is stored in a private repository on HuggingFace.
 
 To download the dataset on the Hub in Python, you need to log in to your Hugging Face account:
 
 ```bash
 huggingface-cli login
```

Comment:
Question, when you have your virtual environment via poetry, you run your programs via “poetry run [bash-command here]”, e..g “poetry run script.py”. Is it the same with this command, e.g. “poetry run huggingface-cli login”? If we want to NOT use the conda environment, we should change this too. If I understand it correctly, when 100% switching to poetry, we will have to preceed every command in the virtual environment with “poetry run” ?

Reply:
So the venv is activated with poetry shell?

Reply:
it basically works like this: poetry creates and handles the venv based on the pyproject files. When you just go to the project folder you always have to type poetry run huggingface_cli or poetry run python path/to/script.py. However, if you do make install or poetry shell (which is contained in make install) you enter the venv. Then you just do huggingface_cli or python path/to/script
Hope this clears it up! Poetry shell is like activating the venv (as an analogy, I'm not 100% what exactly it does, but all basic venv activation features seem to be there and all)

Reply:
Ah sweet, thanks for the clarification!